README.md

```diff
@@ -0,0 +1,51 @@
+thunderbird-flatpak
```

Comment:
I'd just remove it as it's not really relevant on Flathub.

Reply:
Done.